test/simplifyExpression/arithmeticSearch/arithmeticSearch.test.js

```diff
@@ -6,11 +6,11 @@ function testArithmeticSearch(exprStr, outputStr) {
   TestUtil.testSimplification(arithmeticSearch, exprStr, outputStr);
 }
 
-describe.skip('evaluate arithmeticSearch', function () {
+describe('evaluate arithmeticSearch', function () {
```

Comment:
I would also enable `'arithmetic stepping'` in test/simplifyExpression/oneStep.test.js and `'simplify (arithmetic)'` in test/simplifyExpression/simplify.test.js.